packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/errors/DevLauncherUncaughtExceptionHandler.kt

```diff
@@ -84,4 +91,40 @@ class DevLauncherUncaughtExceptionHandler(
       }
     }
   }
+
+  private fun tryToSendExceptionToBundler(exception: Throwable) {
+    if (
+      controller.mode != DevLauncherController.Mode.APP ||
+      !controller.appHost.hasInstance() ||
+      controller.appHost.reactInstanceManager.currentReactContext === null
+    ) {
+      return
+    }
+
+    try {
+      val url = getLogsUrl()
+      val remoteLogManager = DevLauncherRemoteLogManager(DevLauncherKoinContext.app.koin.get(), url)
+        .apply {
+          deferError("Your app just crashed. See the error below.")
+          deferError(exception)
+        }
+      remoteLogManager.sendSync()
+    } catch (e: Throwable) {
+      Log.e("DevLauncher", "Couldn't send an exception to bundler. $e", e)
+    }
+  }
+
+  private fun getLogsUrl(): Uri {
+    val logsUrlFromManifest = controller.manifest?.getRawJson()?.optString("logUrl")
```

Comment:
this should really go through one of the getters in expo-manifest instead of using `getRawJson` (might need to add `getLogUrl` there if it doesn't already exist) because this will still only work for classic update manifests.